package.json

```diff
@@ -203,6 +203,7 @@
     "unified": "^10.1.2",
     "use-sound": "^4.0.1",
     "uuid": "^9.0.0",
+    "valtio": "^2.0.0",
```

Comment:
Seems like we never use this package, are we planning to?

Reply:
I'm removing this in an unpushed commit